### ISSUE TITLE
Sort-Bookmarks stopped working after installing Default Bookmark Folder

### ISSUE THREAD

**Issue by W130SN** [heftig/sort-bookmarks#11]
Sort-Bookmarks stopped working after installing Default Bookmark Folder.
https://addons.mozilla.org/en-GB/firefox/addon/default-bookmark-folder/?src=search
https://github.com/teddy-gustiaux/default-bookmark-folder
After disabling and removing Default Bookmark Folder, Sort-Bookmarks looks busy but doesn't complete the sort.
Tried uninstalling and then reinstalling after restart but still not working. 
Can't see any issue in about:config.
Is there anyway to reset/refresh Sort-Bookmarks to a default state?

**Comment by W130SN** [comment]
Also I tried Auto Sort Bookmarks but that seems to have the same problem.
https://addons.mozilla.org/en-GB/firefox/addon/auto-sort-bookmarks/
https://github.com/eric-bixby/auto-sort-bookmarks-webext

I prefer your addon/extension and it has kept me sane after SortPlaces by Andy Halford was removed.
https://www.andyhalford.com/
https://daniel-lange.com/archives/71-SyncPlaces,-SortPlaces-...-preserving-these-and-other-excellent-Firefox-add-ons.html

Any suggestions to get it working again would be appreciated.
